pkg/sql/sem/tree/backup.go

```diff
@@ -57,7 +66,9 @@ func (node *Backup) Format(ctx *FmtCtx) {
 		ctx.WriteString(" INCREMENTAL FROM ")
 		ctx.FormatNode(&node.IncrementalFrom)
 	}
-	if node.Options != nil {
+
+	var defaultOptions BackupOptions
+	if node.Options != defaultOptions {
```

Comment:
you can compare to empty struct directly, or, maybe even better would be a `IsDefault()` helper?